src/main/java/seedu/address/logic/parser/module/EditModuleCommandParser.java

```diff
@@ -33,16 +33,16 @@ public EditModuleCommand parse(String args) throws ParseException {
         Code code;
 
         try {
+            if (argMultimap.getValue(PREFIX_CODE).isEmpty()) {
+                throw new ParseException("");
+            }
```

Comment:
Should the ParseException be thrown with empty string? No message?

Reply:
Actually I wrote the message for this exception already, but then deleted it because the exception caught on the very next line. Should I comment on why I leave that message empty or just write a message in it?

Reply:
Hmm maybe write the new message on this line then in the following line you can write catch (ParseException pe) { throw pe }

Reply:
actually that thing is on line 40 already